[CS2113-F10-4] Duke of Books

HiIAmTzeKean opened the pull request:
CS students are incredibly busy and hence having a command line interface (CLI) program without GUI makes finding the books they want quick and efficient. This software will also help them track loans and return dates.

imdxz committed:
Merge pull request #4 from ngyongjian/yongjian-AboutUs

Add yj about us

imdxz committed:
Merge branch 'master' into rayden-AboutUs

imdxz committed:
Merge pull request #5 from raydent30/rayden-AboutUs

tp - add rayden.md

HiIAmTzeKean committed:
common Name class

HiIAmTzeKean committed:
book package

HiIAmTzeKean committed:
inventory package

HiIAmTzeKean committed:
person package

HiIAmTzeKean committed:
loan package

HiIAmTzeKean committed:
exception package

HiIAmTzeKean committed:
remove default duke file

HiIAmTzeKean committed:
bug: java files now compile

base classes pushed can now be compiled with gradle

HiIAmTzeKean committed:
base interface

intended to be used by other classes for class compatibility

imdxz committed:
Merge pull request #7 from HiIAmTzeKean/tk-base-model

base models

HiIAmTzeKean committed:
base class with working functionality

Base class are now usable by top modules to perform operations

HiIAmTzeKean committed:
controller to search for book

HiIAmTzeKean committed:
fix: pr8

HiIAmTzeKean committed:
base main file

HiIAmTzeKean committed:
update gradle file for vscode testing

HiIAmTzeKean committed:
feat: inventory controller

controller to handle inventory data

HiIAmTzeKean committed:
test: inventory controller

HiIAmTzeKean committed:
update gitignore

HiIAmTzeKean committed:
Merge pull request #8 from HiIAmTzeKean/tk-book-features

book features

Commit:
Fix format issues

Fixed errors from running `gradle check`.

Commit:
Feature Borrow Book

Commit:
Seperate Books and Loans

raydent30 committed:
30 feat borrow history

raydent30 committed:
30 feat borrow history

HiIAmTzeKean committed:
Merge pull request #12 from danxuZhang/feature-borrow-book

Feature: borrow book

HiIAmTzeKean committed:
Merge pull request #13 from HiIAmTzeKean/10-feat-add-remove-book

10 feat add remove book

HiIAmTzeKean committed:
feat: command package

HiIAmTzeKean committed:
parser package

Commit:
Add LoanController

HiIAmTzeKean committed:
Merge pull request #16 from HiIAmTzeKean/15-feat-top-module-package

15 feat top module package

Commit:
Merge branch 'feature-borrow-book' into 18-feat-loancontroller

imdxz committed:
Merge branch 'AY2223S2-CS2113-F10-4:master' into 18-feat-loancontroller

HiIAmTzeKean committed:
Merge pull request #20 from danxuZhang/18-feat-loancontroller

18 feat loancontroller

raydent30 committed:
Merge branch 'AY2223S2-CS2113-F10-4:master' into 30-feat-book-history

raydent30 committed:
30 feat borrow history - store Loans in Person class

raydent30 committed:
30 feat borrow history - store Loans in Person class

raydent30 committed:
30 feat borrow history - add Loan to ArrayList function, junit tests

imdxz committed:
Merge pull request #14 from raydent30/30-feat-book-history

30 feat book history

Commit:
Add Loan Test

Commit:
Add Loan Controller Test

Commit:
Fix style issues

Commit:
Pass Text UI Test

Commit:
Add dummy test

HiIAmTzeKean committed:
test: search controller tests

HiIAmTzeKean committed:
fix: gradle build style check

HiIAmTzeKean committed:
bug: topic class toString bug

topic string should not be capitalised

HiIAmTzeKean committed:
feat: add callable commands

commands should be sub classes of Command and called by parser.

HiIAmTzeKean committed:
Merge pull request #23 from danxuZhang/22-test-loan-related

22 test loan related

HiIAmTzeKean committed:
Merge branch 'master' into 11-test-search-feat

raydent30 committed:
Merge pull request #24 from HiIAmTzeKean/11-test-search-feat

11 test search feat

imdxz committed:
Merge pull request #25 from HiIAmTzeKean/command

feat: add callable commands

ngyongjian committed:
Update AboutUs - Yongjian

ngyongjian committed:
Merge pull request #1 from ngyongjian/master

Sync latest progress

ngyongjian committed:
top module package - TextUi

Commit:
Change LoanController to static

Commit:
Add get last active loan method

Commit:
Add an abstract LoanCommand

Commit:
Add borrow command and related test

ngyongjian committed:
TextUi class minor edit

ngyongjian committed:
main class implementation

ngyongjian committed:
Parser class prelim version

Commit:
Add return command and related test

Commit:
Add renew command and related test

Commit:
enhance command tests

Commit:
Clean up styles

imdxz committed:
Merge pull request #31 from ngyongjian/15-feat-top-module-package

15 feat top module package

imdxz committed:
Merge branch 'master' into 29-feat-borrow-renew-return

HiIAmTzeKean committed:
feat: search and inventory command

HiIAmTzeKean committed:
Merge branch 'master' into 28-feat-inventory-command

imdxz committed:
Merge pull request #32 from danxuZhang/29-feat-borrow-renew-return

29 feat borrow renew return

raydent30 committed:
add history command

raydent30 committed:
add jUnit assert

raydent30 committed:
add logger in BorrowHistory

Commit:
Add command parser for loan related

Commit:
Add a default user in main

Commit:
Format clean up

Commit:
Add a dummy exit test

Commit:
Fix length bug

ngyongjian committed:
Merge pull request #37 from danxuZhang/36-feat-parser-for-loan-related-command

36 feat parser for loan related command

ngyongjian committed:
Merge pull request #33 from HiIAmTzeKean/28-feat-inventory-command

feat: search and inventory command

imdxz committed:
Merge pull request #34 from raydent30/history-command

add history command

Commit:
Fixed a lot of bugs

raydent30 committed:
Merge pull request #38 from danxuZhang/fix-many-bugs

Fixed a lot of bugs

HiIAmTzeKean committed:
added command to readme

HiIAmTzeKean committed:
feat: search and librarian commands

imdxz committed:
Merge pull request #40 from HiIAmTzeKean/39-bug-search-does-not-work-in-parser-class

search and librarian command

Commit:
Enable assertion

HiIAmTzeKean committed:
doc: search and add

HiIAmTzeKean committed:
Merge pull request #45 from HiIAmTzeKean/44-feat-developer-guide

doc: search and add

HiIAmTzeKean committed:
chore: update gradle build

Accomodate code coverage

HiIAmTzeKean committed:
doc: dg v1.0

imdxz committed:
Merge pull request #49 from HiIAmTzeKean/46-feat-dg-with-v10-functionalities-1

46 feat dg with v10 functionalities

ngyongjian committed:
Merge pull request #43 from danxuZhang/42-enable-assertion

Enable assertion

imdxz committed:
Update DeveloperGuide.md

ngyongjian committed:
Merge pull request #50 from danxuZhang/47-feat-dg

Update DeveloperGuide with Liberian Functions

ngyongjian committed:
Update DeveloperGuide.md

imdxz committed:
Merge pull request #51 from ngyongjian/48-feat-dg-functionalities

Update DeveloperGuide.md

HiIAmTzeKean committed:
docs: search and librarian docs

HiIAmTzeKean committed:
feat: inclusion of class diagram

imdxz committed:
Merge pull request #55 from HiIAmTzeKean/dg

Developer guide

Commit:
Create user class

Commit:
cleanup

Commit:
Add Access Controller

Commit:
cleanup

chongyongrui commented on docs/DeveloperGuide.md:
```diff
@@ -2,32 +2,143 @@
+- search -title Python Programming  
+For successful searches, the program will output the relevant book along with the book details: ISBN, Title, Author, Topic. For unsuccessful searches, the program will output a string informing the user that there is no match with the input title from the inventory.
+
+Note: Partial string matching will not be considered in this application. Exact match of title and topic will be done, and if match is found, the book object will be returned.
```
I like how you add additional notes to make it clearer to the reader about the more specific input cases

chongyongrui commented on docs/DeveloperGuide.md:
```diff
@@ -2,32 +2,143 @@
 ## Non-Functional Requirements
 
-{Give non-functional requirements}
+1. Data should be stored in text file so that information like borrowed books and inventory are not lost
```
Perhaps you could create a sequence diagram to show how existing data is read and parsed and stored in a class to be edited and changed, and then how the new updated data is saved

chongyongrui commented on docs/DeveloperGuide.md:
```diff
@@ -2,32 +2,143 @@
+
+![](images/Search%20class%20diagram.png)
+Description: Searches for a book using the title  
+Format: search -title TITLE
```
perhaps when sharing the input example, you could use the `(text)` to make the DG more readable

chongyongrui commented on docs/DeveloperGuide.md:
```diff
@@ -2,32 +2,143 @@
+This feature is only applicable for admin. For successful addition of new book, the program will output a message to inform the librarian. The new book will also be added into the inventory of the system so that users can now borrow the book from the library. Error inputs and missing inputs will be handled accordingly.
+ 
+
+### [Proposed] Delete book
```
perhaps you can create a sequence diagram for each of these command steps to share how the data is changed when such commands are used

chongyongrui commented on docs/DeveloperGuide.md:
```diff
@@ -2,32 +2,143 @@
+
+### [Proposed]  Search book by topic
+Description: Searches for a book by its topic  
+Format: search -topic TOPIC
```
this feels more like what the UG should include, not the DG

Commit:
Separate User Command and Access Command

Separate commands into user command and access command.
Access command used for login the system, user command used for user
actions.

Commit:
Seprate Parser for Access and User Commands

Commit:
Add Login Logout function

Commit:
Add multiple words login details

Commit:
Add change password

Commit:
Format and cleanup

Commit:
Pass Text UI Test

Commit:
Fix typo

bentohset commented on docs/DeveloperGuide.md:
```diff
@@ -2,32 +2,143 @@
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+All commands will be fetched by the getUserCommand() method in the TextUi class. The parseCommand() method in the Parser class will then parse the command and pass it to the respective command classes. The command classes will be responsible for executing the relevant features.
+
+### [Proposed] Search book by title
+
+![](images/Search%20class%20diagram.png)
```
Seems to be some formatting error on the DG website
![bug](https://user-images.githubusercontent.com/115091471/228720288-527fd2a7-28a4-4995-abc5-34dbebeec3e9.JPG)

tyuyang replied:
Since you are documenting a specific feature, a sequence diagram could be more suitable rather than a class diagram. Furthermore, the class diagram seems to have formatting errors. 
![image](https://user-images.githubusercontent.com/88535855/228740083-187315fd-d03b-46f4-bf84-db921edf656e.png)
The format for attributes should be : `visibility name: type`. So in your case, the attribute should be `- inventory: Inventory`. 
Similarly, format for methods should be `visibility name(parameters): return type`. So in your case, the first method should look like this `+ setData(dataPoint: T)`. Note that for template parameters, refer to [this](https://www.uml-diagrams.org/template.html) for the format.

bentohset commented on docs/DeveloperGuide.md:
```diff
@@ -2,32 +2,143 @@
+Format: borrow -title TITLE
+Example:
+- borrow -title Python Programming  
+For successful borrowing, the program will output a string showing that the action is successful and, at the same time, mark the book as borrowed in the system. For unsuccessful borrow requests, the program will either output that there is no such book in the inventory or a message showing that the book is already on loan at the time of the borrow request.
```
Perhaps you could include more technical documentation on how the command runs algorithmically.

Same goes for the rest of the features! That way it will feel more like a DG instead of a UG.

tyuyang commented on docs/DeveloperGuide.md:
```diff
@@ -2,32 +2,143 @@
-|v1.0|new user|see usage instructions|refer to them when I forget how to use the application|
-|v2.0|user|find a to-do item by name|locate a to-do without having to go through the entire list|
+|v1.0|new user|be able to search for a book|borrow a book to read|
+|v1.0|libriarian|add or remove book|change the books in the inventory|
+|v1.0|user|return book i borrowed|remove the borrow status taged to my account|
+|v2.0|libriarian|pay my fine|continue to borrow more books and see my exam results|
```
Do ensure that you check for spelling mistakes. Librarian is the correct spelling

tyuyang commented on docs/DeveloperGuide.md:
Perhaps there should be more information on how the program runs. The DG is supposed to be for future developers after all. Things like system architecture, component relations and dependencies should be inside

tyuyang commented on docs/DeveloperGuide.md:
```diff
@@ -2,32 +2,143 @@
+### [Proposed] Search book by title
+
+![](images/Search%20class%20diagram.png)
+Description: Searches for a book using the title  
+Format: search -title TITLE
+Example:
+- search -title Python Programming  
+For successful searches, the program will output the relevant book along with the book details: ISBN, Title, Author, Topic. For unsuccessful searches, the program will output a string informing the user that there is no match with the input title from the inventory.
```
You should leave one more line between line 16 and 17 or specify a line break `<br />`. Cos currently it looks like this:
![image](https://user-images.githubusercontent.com/88535855/228737029-2539d9b1-515e-4862-9076-9227123c2f71.png)

As you can see, the Format and Example is on the same line, which doesn't seem to be intentional.

tyuyang commented on docs/DeveloperGuide.md:
```diff
@@ -2,32 +2,143 @@
+The program will list out all the books in the library inventory. If the inventory is empty, the program will output a message indicating that there is currently no book in the library system.
+
+
+### [Proposed] View history
+Description: Check the borrow history of the books (admin and user versions). The admin version will allow admin to access the whole borrow history, while users can only view their own borrow history.  
+Format: history
+Example:
+- history  
+The program will output the borrow history of books according to the accessibility as mentioned in the description above. The output includes the book title and other book details, the status of the book (either it is available for borrowing or it has been borrowed at the moment), and also details of the loan, which include the date and time of borrowing and returning.
```
It will be helpful to include which classes are involved in this feature. Using a sequence diagram is probably a good way to do that.

HiIAmTzeKean committed:
Merge pull request #58 from danxuZhang/35-feat-user-login-and-logout

Add Login Logout

Commit:
Add Access Management User Guide

HiIAmTzeKean committed:
feat: search multiple books

search now returns multiple matches

HiIAmTzeKean committed:
test: search test cases

test case should also factor in return of arraylist

ngyongjian committed:
Merge pull request #63 from HiIAmTzeKean/59-feat-search-command-to-work-with-title-and-topic

59 feat search command to work with title and topic

ngyongjian committed:
Update DeveloperGuide.md

raydent30 committed:
Merge pull request #65 from ngyongjian/64-update-dg

Update DeveloperGuide.md

Commit:
Update UG

raydent30 committed:
add .txt database for books
fix search by title & topic bug

raydent30 committed:
edit check style

theopin commented on src/main/java/seedu/dukeofbooks/parser/UserCommandParser.java:
```diff
@@ -0,0 +1,245 @@
+
+        if (titleIndex == -1 && topicIndex == -1 && authorIndex == -1
+                && actionIndex == -1 && actionIndex == -1) {
+            return new IncorrectUserCommand(MESSAGE_INVALID_COMMAND_FORMAT);
```
-1 seems to be used quite commonly in this method as a magic literal, perhaps consider renaming it as a constant variable for better readability?

theopin commented on src/main/java/seedu/dukeofbooks/parser/UserCommandParser.java:
```diff
@@ -0,0 +1,245 @@
+            String author = getWord(authorIndex, parts);
+            String action = getWord(actionIndex, parts);
+            String isbn = getWord(isbnIndex, parts);
+            // String isbn = Book.createISBN();
```
Stray line code, do remember to remove it if it is not going to be used

theopin commented on src/main/java/seedu/dukeofbooks/ui/TextUi.java:
```diff
@@ -0,0 +1,108 @@
+    private static String getIndexedListForViewing(List<String> listItems) {
+        final StringBuilder formatted = new StringBuilder();
+        int displayIndex = DISPLAYED_INDEX_OFFSET;
+        for (String listItem : listItems) {
+            formatted.append(getIndexedListItem(displayIndex, listItem)).append("\n");
+            displayIndex++;
```
Perhaps consider renaming it to a constant or utilize previously defined LS if behaviour is the same?

theopin commented on src/main/java/seedu/dukeofbooks/ui/TextUi.java:
```diff
@@ -0,0 +1,108 @@
+public class TextUi {
+    public static final int DISPLAYED_INDEX_OFFSET = 1;
+    private static final String LINE_PREFIX = "|| ";
+    private static final String LS = System.lineSeparator();
```
Best to spell out this variable rather than using abbreviations (eg. LINE_SEPARATOR)

theopin commented on src/main/java/seedu/dukeofbooks/DukeOfBooks.java:
```diff
@@ -0,0 +1,140 @@
+            authorPhone = new Phone(87654321);
+            author = new Person(authorName, authorPhone);
+            book = new Book(isbn, title, topic, author);
+            InventoryController.addBook(book);
```
try not to reuse variables. Do consider either abstracting this behaviour to another method or using new variable names for such code

theopin commented on src/main/java/seedu/dukeofbooks/data/book/BorrowHistory.java:
```diff
@@ -0,0 +1,43 @@
+                count++;
+            }
+        }
+        // logger.log(Level.INFO, "Finished processing checkHistory class!");
```
Do remember to remove out commented code if they are not going to be used

theopin commented on src/main/java/seedu/dukeofbooks/data/book/BorrowHistory.java:
```diff
@@ -0,0 +1,43 @@
+                }
+                output = output + count + ". " + borrowedItem.toString();
+                output = output + " Status: " +
+                        (borrowedItem.isBorrowed ? "borrowed" : "returned");
```
try to avoid reusage of variables. Assign the output to a new variable instead

raydent30 committed:
Merge pull request #116 from raydent30/txt-database-for-books

book database & search bug

theopin reviewed (commented):
Left some comments to improve your code quality! All the best for your implementation!

Commit:
Change Access Command Return Type

Commit:
Add Prompt When User Executes Exit

imdxz committed:
Merge branch 'master' into 62-incorrect-return-type-for-access-command

Commit:
Pass checkstyle

Commit:
Store user data

Commit:
Pass Text UI Test

ngyongjian committed:
Merge pull request #120 from danxuZhang/62-incorrect-return-type-for-access-command

62 incorrect return type for access command

ngyongjian committed:
Merge branch 'master' into 119-no-user-data

ngyongjian committed:
Merge pull request #121 from danxuZhang/119-no-user-data

Store user data

ngyongjian committed:
Implement payment function for overdue loans

ngyongjian committed:
Minor fixes to meet input requirements

ngyongjian committed:
remove unused imports

ngyongjian committed:
Add throw declaration

ngyongjian committed:
Fix checks

ngyongjian committed:
Fix checks

ngyongjian committed:
fix checks

ngyongjian committed:
fix checks

ngyongjian committed:
fix checks

ngyongjian committed:
Merge pull request #2 from ngyongjian/122-feat-payment-function

122 feat payment function

ngyongjian committed:
feat: Check item availability

ngyongjian committed:
fix checks

ngyongjian committed:
Merge pull request #3 from ngyongjian/124-feat-check-book-availability

Update 126

ngyongjian committed:
feat: Check borrowing status

ngyongjian committed:
fix checks

HiIAmTzeKean committed:
doc: fixed issue of wrong example

HiIAmTzeKean committed:
bug: fixed buggy add action

Add action should not be defaulted to delete.

HiIAmTzeKean committed:
chore: fixed code style

ngyongjian committed:
Merge pull request #127 from ngyongjian/126-feat-check-borrowing-status

126 feat check borrowing status

raydent30 committed:
Merge branch 'master' into 107-add-bug

raydent30 committed:
Merge pull request #128 from HiIAmTzeKean/107-add-bug

fixed librarian add command

HiIAmTzeKean committed:
Merge pull request #61 from danxuZhang/60-login-logout-ug

Add Access Management User Guide

Commit:
Clear User Database

raydent30 committed:
add list command implementation

HiIAmTzeKean committed:
Merge pull request #130 from raydent30/ft-13-List-book

add list command implementation

ngyongjian committed:
Update README.md

ngyongjian committed:
Add AboutUs images

ngyongjian committed:
Update AboutUs images

ngyongjian committed:
Update AboutUs images

ngyongjian committed:
Update AboutUs.md

ngyongjian committed:
Update DeveloperGuide.md

ngyongjian committed:
Update README.md

ngyongjian committed:
Update UserGuide.md

ngyongjian committed:
Update UserGuide.md

ngyongjian committed:
fix bug for check borrowing status

ngyongjian committed:
fix gradle checks

ngyongjian committed:
fix gradle checks

Commit:
Fix cannot reload program bug

Commit:
Add Due Date to Borrow History

imdxz committed:
Merge branch 'master' into 129-bug-access-management

Commit:
Update DG

HiIAmTzeKean committed:
Merge pull request #134 from ngyongjian/133-doc-update-all-documentations

133 doc update all documentations

HiIAmTzeKean committed:
Merge branch 'master' into doc-add-access-management-dg

HiIAmTzeKean committed:
Merge pull request #140 from danxuZhang/doc-add-access-management-dg

Update DG for AccessCommands

HiIAmTzeKean committed:
chore: tidy up missing sentences

HiIAmTzeKean committed:
doc: adjustment of paragraph format

HiIAmTzeKean committed:
doc: alignment of TOC

HiIAmTzeKean committed:
doc: truncating list example

ngyongjian committed:
Merge pull request #136 from ngyongjian/135-bug-check-borrowing-status-date-format-error

fix bug for check borrowing status

ngyongjian committed:
Merge pull request #142 from HiIAmTzeKean/documentation-fix

Documentation fix

ngyongjian committed:
Merge pull request #137 from danxuZhang/129-bug-access-management

129 bug access management

ngyongjian committed:
Merge pull request #139 from danxuZhang/132

Add Due Date to Borrow History

ngyongjian committed:
change file name

ngyongjian committed:
Update ngyongjian.md

ngyongjian committed:
Update ngyongjian.md

ngyongjian committed:
Merge pull request #144 from ngyongjian/143-doc-yongjian-ppp

143 doc yongjian ppp

ngyongjian committed:
Update AboutUs.md

ngyongjian committed:
Add "" to img src to allow display in web format

HiIAmTzeKean committed:
feat: plantuml diagram

HiIAmTzeKean committed:
doc: update diagrams

Commit:
Add Danxu PPP

Commit:
Rename Danxu PPP

imdxz committed:
Merge pull request #150 from danxuZhang/149-add-danxu-ppp

Add Danxu PPP

raydent30 committed:
add rayden ppp

HiIAmTzeKean committed:
Merge pull request #151 from raydent30/rayden-ppp

add rayden ppp

HiIAmTzeKean committed:
Merge branch 'master' into ngyongjian-patch-1

HiIAmTzeKean committed:
Merge pull request #145 from AY2223S2-CS2113-F10-4/ngyongjian-patch-1

Update ppp link in AboutUs.md

HiIAmTzeKean committed:
Merge branch 'master' into ngyongjian-patch-2

HiIAmTzeKean committed:
Merge pull request #147 from AY2223S2-CS2113-F10-4/ngyongjian-patch-2

Add "" to img src to allow display in web format

raydent30 committed:
Merge pull request #154 from HiIAmTzeKean/dg-diagram

Dg diagram

HiIAmTzeKean committed:
feat: TzeKean ppp added feature

HiIAmTzeKean committed:
bug: fixed about us table

HiIAmTzeKean committed:
feat: updated pr links to ppp

HiIAmTzeKean committed:
doc: fixed broken link

raydent30 committed:
edit documentations

HiIAmTzeKean committed:
test: inventory controller

HiIAmTzeKean committed:
Merge pull request #156 from raydent30/documentation-qc

edit documentations

HiIAmTzeKean committed:
feat: FAQ common questions

HiIAmTzeKean committed:
doc: update search note

search no longer only returns words with exact string matching.

HiIAmTzeKean committed:
chore: rename file

PPP should be using my github name

HiIAmTzeKean committed:
doc: about us

HiIAmTzeKean committed:
bug: payment method

HiIAmTzeKean committed:
feat: message helper for invalid syntax

HiIAmTzeKean committed:
fix: payment overdue issue

HiIAmTzeKean committed:
bug: table render bug

HiIAmTzeKean committed:
bug: code style

HiIAmTzeKean committed:
bug: display block style on image

HiIAmTzeKean committed:
doc: testing table format

HiIAmTzeKean committed:
add bars

imdxz committed:
Merge pull request #162 from HiIAmTzeKean/dg-search

doc: update search note

imdxz committed:
Merge pull request #161 from HiIAmTzeKean/ug-faq

feat: FAQ common questions

imdxz committed:
Merge pull request #157 from HiIAmTzeKean/tk-ppp

TzeKean ppp

HiIAmTzeKean committed:
feat: image renders!

raydent30 committed:
Merge pull request #163 from HiIAmTzeKean/aboutus

doc: about us

HiIAmTzeKean committed:
doc: update jar name

HiIAmTzeKean committed:
remove files

files like jar and database should not be in root

HiIAmTzeKean committed:
feat: create default dir and file

HiIAmTzeKean committed:
feat: guide in pdf version

raydent30 committed:
Merge pull request #165 from HiIAmTzeKean/aboutus-table

Aboutus table

raydent30 committed:
Merge pull request #167 from HiIAmTzeKean/aboutus-table

bug: display block style on image

HiIAmTzeKean committed:
feat: update tk PPP

raydent30 committed:
Merge pull request #169 from HiIAmTzeKean/aboutus-table

doc: testing table format

imdxz committed:
Merge pull request #170 from HiIAmTzeKean/aboutus-table-working

add bars to aboutus

Commit:
Update DG UG

raydent30 committed:
Merge pull request #172 from HiIAmTzeKean/aboutus-table-working

feat: image renders!

imdxz committed:
Merge branch 'master' into 166-doc-update-dg-ug

raydent30 committed:
Merge pull request #158 from HiIAmTzeKean/inventory-test

test: inventory controller

HiIAmTzeKean committed:
Merge pull request #171 from danxuZhang/166-doc-update-dg-ug

Update DG UG

raydent30 committed:
Merge pull request #164 from HiIAmTzeKean/payment-feat

Payment feat

raydent30 committed:
Merge pull request #173 from HiIAmTzeKean/final-submission

create default dir and file

raydent30 committed:
Merge pull request #174 from HiIAmTzeKean/pdf-guides

feat: guide in pdf version

HiIAmTzeKean committed:
Merge pull request #176 from HiIAmTzeKean/pdf-guides

feat: update tk PPP

HiIAmTzeKean committed:
feat: renew question

HiIAmTzeKean committed:
feat: update ug pdf

imdxz committed:
Merge pull request #177 from HiIAmTzeKean/ug-qna

feat: renew question in UG
